entities/src/main/resources/init-db.sql

```diff
@@ -3,6 +3,8 @@ INSERT INTO album (id, name, year) VALUES (1, 'Volume One on One', 2002), (2, 'B
 INSERT INTO artist_album (artist_id, album_id) VALUES (1, 1), (2, 1), (3, 2);
 INSERT INTO song (id, name, duration, listens) VALUES (1, 'Sky and Sand', 239, 0), (2, 'Altes Kamuffel', 275, 0), (3, 'Too Many Answers', 172, 0);
 INSERT INTO artist_song (artist_id, song_id) VALUES (3, 1), (3, 2), (4, 1), (5, 3);
+INSERT INTO playlist (id, name) VALUES (1, 'Prijetne vize za lepsi vsakdan');
```

Comment:
When adding new items in `init-db.sql`, you need to also reset the ID of that entity afterwards. Otherwise (as in this case), a duplication error is thrown when trying to add a new playlist as the system tries to add another playlist with ID=1.

To fix this, add an `ALTER SEQUENCE playlist_id_seq RESTART WITH <number>` as is already done for other entities below.

Reply:
Amazing review Matej! Thank you for this `ALTER SEQUENCE` snippet, I did not know that.